substrate/frame/revive/src/tests.rs

```diff
@@ -374,14 +374,15 @@ impl RegisteredChainExtension<Test> for TempStorageExtension {
 parameter_types! {
 	pub BlockWeights: frame_system::limits::BlockWeights =
 		frame_system::limits::BlockWeights::simple_max(
-			Weight::from_parts(2u64 * WEIGHT_REF_TIME_PER_SECOND, u64::MAX),
+			Weight::from_parts(2 * WEIGHT_REF_TIME_PER_SECOND, u64::MAX),
 		);
```

Comment:
can't we delete this and just use the default 
https://github.com/paritytech/polkadot-sdk/blob/pg/fix-gas-encoding/substrate/frame/system/src/limits.rs?plain=1#L208

Reply:
Yeah I think we should have this consistent. But I think it should be the maximum we aim to support. What would that be?

Reply:
We should tune to 2s: https://wiki.polkadot.network/docs/maintain-guides-async-backing#phase-3---activate-async-backing